packer/windows.pkr.hcl

```diff
@@ -0,0 +1,209 @@
+variable "gcp_project" { type = string }
+variable "task_name" { type = string }
+
+variable "build_type" {
+  type = string
+  default = "googlecompute"
+}
+
+variable "docker_repo" {
+  type = string
+  default = ""
+}
+
+variable "docker_server" {
+  type = string
+  default = ""
+}
+
+variable "gcp_password" {
+  type = string
+  default = ""
+}
+
+variable "image_date" {
+  type = string
+  default = ""
```

Comment:
I don't think most of these should have a default value - the packer file won't build without them set...

Reply:
Yes, since 'windows.pkr.hcl' used by both vm and docker image generation; I didn't want to change packer commands.